src/shared/header.mustache

```diff
@@ -61,11 +62,11 @@
             <span class="sr-only">Menu</span>
             <span class="navbar-toggler-icon" aria-hidden="true"></span>
         </button>
-        <a class="navbar-brand" href="/">
+        <a class="navbar-brand" href="https://nijmegen.nl">
```

Comment:
geen www.nijmegen.nl?

Reply:
Ik heb blijkbaar een persoonlijke voorkeur om www te redirecten naar het 'kale' domein en die dus ook overal te gebruiken. Ik zie dat we het andersom doen, dus naar www. kan dan natuurlijk ook.